src/cmd.js

```diff
@@ -103,6 +95,7 @@ function _cmd(options, command, commandArgs, userOptions) {
   var defaultOptions = {
     maxBuffer: DEFAULT_MAXBUFFER_SIZE,
     stripEof: false, // Preserve trailing newlines for consistency with unix.
```

Comment:
You can delete the `stripEof` line (`stripEof` was renamed to `stripFinalNewline` in execa 2.0). Move the "Preserve trailing newlines..." comment to the same line as stripFinalNewline please.